### PR TITLE
adding foodlist

### PR DESCRIPTION
**Pull request by FooChiHen**
TODO adding next PR to include db queries

mounted foodlist under user endpoint because we can get the user ID from url parameters. do correct me if i am wrong tho

### CODE REVIEW

File: src/services/playlist.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+			FoodName: "hehe",
+		},
+	}
+	render.JSON(w, r, FoodlistServices)
+}
+
+func (*FoodlistService) GetFoodlist(w http.ResponseWriter, r *http.Request) {
+	FoodlistServices := []models.Foodlist{}
+
+	id := r.Context().Value("FoodlistServiceID").(string)
```
Comment (mahe993) [inline src/services/playlist.go]: i think the context key used was `"foodlist"`?  Anyway, we dont have to use ctx if you feel its unnecessary. The one in user.go is just an example. Our project probably has no usecase for ctx its too small, but up to you! im fine with this

**Review by mahe993 — DISMISSED** [review]
Personally, i prefer to separate it out into foodlist.go rather than updating user.go. The endpoint cld be `foodlist/{userID}/{id}`

Lets rename the files to be `foodlist.go` as well. Otherwise looks good! thank you

**Review by mahe993 — DISMISSED** [review]

File: foodloop-db-files/01_schema.sql [hunk 2]
```diff
@@ -1,6 +1,6 @@
-DROP DATABASE IF EXISTS foodloopdb;
-CREATE DATABASE foodloopdb;
-\c foodloopdb;
+-- DROP DATABASE IF EXISTS foodloopdb;
+-- CREATE DATABASE foodloopdb;
+-- \c foodloopdb;
```
Comment (mahe993) [inline foodloop-db-files/01_schema.sql]: is this intended?

File: foodloop-db-files/seeders/02_insertFood.sql [hunk 3]
```diff
@@ -1,3 +1,4 @@
+BEGIN;
```
Comment (mahe993) [inline foodloop-db-files/seeders/02_insertFood.sql]: what does this and `COMMIT;` at the EOF do?

File: foodloop-db-files/01_schema.sql (hunk 2 above)
Comment (FooChiHen) [reply foodloop-db-files/01_schema.sql]: might possibly be caused by linter?

File: foodloop-db-files/seeders/02_insertFood.sql (hunk 3 above)
Comment (chae-seong) [reply foodloop-db-files/seeders/02_insertFood.sql]: this is not needed, will remove it

File: foodloop-db-files/01_schema.sql (hunk 2 above)
Comment (chae-seong) [reply foodloop-db-files/01_schema.sql]: have uncommented these lines and added `\c postgres` in Line 1 to switch connection to the `postgres` database before attempting to drop `foodloopdb`

this way, we would no longer be connected to `foodloopdb` while attempting to drop it

**Review by chae-seong — APPROVED** [review]

**Review by mahe993 — APPROVED** [review]

**Pull request merged**